packages/components/pages/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -135,6 +135,14 @@ module.exports = class Checkout extends Page {
             get error () { return $(FIELDS.addressAdministrativeArea.error); }
         }
     }
+
+    get customerInput () { return this.customerInfo || 'customer info has not been set'; }
+
+    set expectedCustomerInfo (info) {
+        this.customerInfo = info;
+    }
```

Comment:
There are a few things here that are confusing to me:

1. You have a `getter` called `customerInput` but it returns an `object` (most of the times 😬). In the lines above, when we're talking about inputs, we are referring to an input field (e.g. `text`).
2. Your getter returns an `object` (if you have one), otherwise a `String`. Normally you should try to return the same type in a function/method/property. So, instead, I'd return an empty `object`. The problem, otherwise, is that the consumers of this `getter` need to worry about handling different types of results. Makes sense? 😄
3. You can (and normally should, I believe) use the same name for the `getter` and the `setter`.